ocpmodels/datasets/trajectory_lmdb.py

```diff
@@ -99,6 +100,8 @@ def connect_db(self, lmdb_path=None):
             readonly=True,
             lock=False,
             readahead=False,
+            meminit=False,
+            max_readers=1,
         )
         return env
 
```

Comment:
Add a `close()` method to the dataset class that closes all lmdb files. Training loop can call this method at the end of training.